api/meetups/migrations/0001_initial.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.1.2 on 2019-03-05 16:28
```

Comment:
Question, should migration file be pushed or ignored ? I am having an issue with pushing migrations file. I feel everyone should add it to gitignore. Please clarify this for me

Reply:
Hello @Bakley ,
It is okay to push the migrations file, so when one gets the repo, they just run `python manage.py migrate`
and only make migrations if they have altered the models or have models whose migrations were not made(Django will notify you in this case)

Reply:
Ooh, in that case its fine. I was worried it might bring errors or merge conflict